scripts/rqt_graph

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Use an entry point instead in order to create an executable on Windows.

Reply:
This script file has been modified and moved to src/rqt_graph/main.py and an entry point has been created in setup.py.